add temporal fix for print_report() (#261)
epiverse-trace/cleanepi: NEWS.md

## Bug fixes

* `replace_missing_values()` now matches with `na_strings` insensitive to
case or leading/trailing white space (#257, @joshwlambert).

* Fix {cli}-related errors in `check_subject_ids()` function(#260, @Karim-Mane).

> * fix error in `print_report()` by deduplicating the elements of the report

Reviewer: ```suggestion
* fix error in `print_report()` by remove duplicated elements from the report
```
